src/manifest.ts

```diff
@@ -22,7 +22,7 @@ export async function readManifest(): Promise<Manifest> {
   if (typeof rootDir === 'undefined') rootDir = DOT.PROJECT.DIR;
   const manifest = path.join(rootDir, PROJECT_MANIFEST_FILENAME);
   try {
-    return fs.readJsonSync(manifest, { encoding: 'utf-8' });
+    return fs.readJsonSync(manifest, { encoding: 'utf8' });
```

Comment:
Please use or create a constant for `'utf8'`.

Reply:
@grant just to make sure: creating a `globals.ts` to hold `export const UTF8 = 'utf8'` is what you have in mind?

Reply:
Create a new constant in `files.ts`:

```ts
// @see https://nodejs.org/api/fs.html#fs_fs_readfilesync_path_options
export const FS_OPTIONS = { encoding: 'utf8' };
```

Use in the `src/` files:

https://github.com/google/clasp/search?q=utf&unscoped_q=utf

You don't have to use it in the test file.

Reply:
Done